components/layout/display_list/builder.rs

```diff
@@ -2012,6 +2012,7 @@ impl Fragment {
                         offset: LayoutVector2D::new(shadow.horizontal.px(), shadow.vertical.px()),
                         color: self.style.resolve_color(shadow.color).to_layout(),
                         blur_radius: shadow.blur.px(),
+                        should_inflate: false,
```

Comment:
This should be true (Gecko inflates shadow bounds already to account for the blurs, Servo does not).